### DISCUSSION TITLE
override config key at command line

### DISCUSSION THREAD

**Discussion by lstrgar** [facebookresearch/hydra#2428] [Q&A] [answered]
Hi. I'm surprised to see this does not work:

main.yaml
```
defaults:
  - _self_
  - optim: adam
```

optim/adam.yaml
```
lr: 0.001
betas: [0.9, 0.999]
```

`python run.py optim.lr=0.01`

When I run this above, the original value of 0.001 is still being set. What am I missing?

**Comment by Jasha10** [comment, marked as answer]
Not sure. I'm not able to reproduce the behavior you're seeing; when I run `python run.py optim.lr=0.01` on my machine, I get `{'optim': {'lr': 0.01, 'betas': [0.9, 0.999]}}`.

Could you paste the output of the following command?
```text
python run.py optim.lr=0.01 --info
```

**Reply by lstrgar** [reply]
You are right. I had a bug where loading a model checkpoint was overwriting values read from the config.
